dds_web/api/user.py

```diff
@@ -949,6 +949,23 @@ def get(self):
                 )
             )
 
+        if auth.current_user().role == "Super Admin":
+            json_input = flask.request.json
+            if not json_input:
+                raise ddserr.DDSArgumentError(message="Unit public id missing.")
+
+            unit = json_input.get("unit")
+            if not unit:
+                raise ddserr.DDSArgumentError(message="Unit public id missing.")
```

Comment:
Not sure that it's better than the current approach, but at least it's not repeating the same error message:
```suggestion
            if not json_input or not (unit := json_input.get("unit")):
                raise ddserr.DDSArgumentError(message="Unit public id missing.")
```